athena-elasticsearch/src/main/java/com/amazonaws/connectors/athena/elasticsearch/ElasticsearchMetadataHandler.java

```diff
@@ -179,9 +194,6 @@ public ListTablesResponse doListTables(BlockAllocator allocator, ListTablesReque
             catch (IOException error) {
                 throw new RuntimeException("Error retrieving indices: " + error.getMessage(), error);
             }
-            finally {
-                client.shutdown();
```

Comment:
who/what is closing clients if this is removed everywhere?

Reply:
Clients are closed by the client cache upon eviction from the cache.